don't install pre-compiled 'bedtools' binary, since BEDTools is provided as a proper dependency via EasyBuild
author: Kenneth Hoste (HPC-UGent)
--- ngs-smap-4.6.5/setup.cfg.orig	2022-10-11 09:59:22.522325500 +0200
+++ ngs-smap-4.6.5/setup.cfg	2023-04-07 16:48:58.434620448 +0200
@@ -49,10 +49,6 @@
 	sphinx_rtd_theme~=1.0.0
 	sphinx-tabs~=3.2.0
 
-[options.data_files]
-bin = 
-	bin/bedtools
-
 [egg_info]
 tag_build = 
 tag_date = 0
